### PR TITLE
BRD - The Lyceum

### PR DESCRIPTION
**Pull request by Tobschinski**
+ added alternating spawns of Shadowforge Flame Keeper
+ added missing Anvilrage Reservists
+ grouped Anvilrage Reservists
+ added 3 patrolling groups

### CODE REVIEW

File: sql/migrations/20180519193248_world.sql [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+SET @CGUID := 53301;
+
+-- --------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+DELETE FROM `creature` WHERE `id`=8901 AND `map`=230 AND NOT `guid` IN (45955, 45956, 45958, 45959); -- at General Angerforge
+DELETE a FROM `creature_movement` a INNER JOIN `creature` b ON a.`id`=b.`guid` WHERE b.`id`=8901;
+DELETE a FROM `creature_addon` a INNER JOIN `creature` b ON a.`guid`=b.`guid` WHERE b.`id`=8901;
+
+REPLACE INTO `creature` (`guid`, `id`, `map`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `DeathState`, `MovementType`, `spawnFlags`, `visibilitymod`, `patch_min`, `patch_max`) VALUES
+(@CGUID + 000, '8901', '230', '0', '0', '1414.053', '-462.9212', '-91.92727', '3.52556500', '30', '30', '0', '0', '883', '0', '0', '0', '0', '0', '0', '10'),
```
Comment (Kittnz) [inline sql/migrations/20180519193248_world.sql:20]: Data comes from?

Comment (Tobschinski) [reply sql/migrations/20180519193248_world.sql:20]: wrath

Comment (Kittnz) [reply sql/migrations/20180519193248_world.sql:20]: sniffs? or

Comment (Tobschinski) [reply sql/migrations/20180519193248_world.sql:20]: yes sir

**Pull request merged**